app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type annotations for all function parameters and the return type. In this case, you should specify the type of the `cafe` parameter. This is related to the 'Use annotation, it is a good practice' point in the checklist.